lib/models/sponsors_data.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:ecellapp/models/sponsor.dart';
```

Comment:
This model is not useful imo. Do the following
1. Make a `SponsorCategory` model with fields `String name` and `List<Sponsor> sponsors`, and `fromJson(Map<String, dynamic> data)` constructor.
2. Create a `List<SponsorCategory> sponsorsData` variable and store all the data in this variable. Run a `forEach` on `spons_category` and for each category and add the `SponsorCategory` to the `sponsorsData` list.